### PR TITLE
Use Literal in a few more places

### PR DESCRIPTION
**Pull request by srittau**
Prefer Literal import from typing, if available

### CODE REVIEW

File: stdlib/3/_winapi.pyi [hunk 1]
```diff
@@ -46,11 +52,10 @@ WAIT_TIMEOUT: int
 
 def CloseHandle(handle: int) -> None: ...
 
-# TODO: once literal types are supported, overload with Literal[True/False]
 @overload
-def ConnectNamedPipe(handle: int, overlapped: Union[int, bool]) -> Any: ...
+def ConnectNamedPipe(handle: int, overlapped: Literal[True]) -> Overlapped: ...
 @overload
-def ConnectNamedPipe(handle: int) -> None: ...
+def ConnectNamedPipe(handle: int, overlapped: Literal[False] = ...) -> None: ...
```
Comment (JelleZijlstra) [inline stdlib/3/_winapi.pyi]: Also need an overload for just `bool`

File: third_party/2and3/werkzeug/wrappers.pyi [hunk 2]
```diff
@@ -44,8 +49,12 @@ class BaseRequest:
...
     @property
     def data(self) -> bytes: ...
-    # TODO: once Literal types are supported, overload with as_text
-    def get_data(self, cache: bool = ..., as_text: bool = ..., parse_form_data: bool = ...) -> Any: ...  # returns bytes if as_text is False (the default), else Text
+    @overload
+    def get_data(self, cache: bool = ..., as_text: Literal[False] = ..., parse_form_data: bool = ...) -> bytes: ...
+    @overload
+    def get_data(self, cache: bool, as_text: Literal[True], parse_form_data: bool = ...) -> Text: ...
+    @overload
+    def get_data(self, *, as_text: Literal[True], parse_form_data: bool = ...) -> Text: ...
```
Comment (JelleZijlstra) [inline third_party/2and3/werkzeug/wrappers.pyi]: Same here; we need an overload on `bool` for the case where the type checker can't infer the value.

File: third_party/2and3/werkzeug/wrappers.pyi [hunk 3]
```diff
@@ -107,8 +116,10 @@ class BaseResponse:
     def force_type(cls: Type[_SelfT], response: object, environ: Optional[WSGIEnvironment] = ...) -> _SelfT: ...
     @classmethod
     def from_app(cls: Type[_SelfT], app: Any, environ: WSGIEnvironment, buffered: bool = ...) -> _SelfT: ...
-    # TODO: once Literal types are supported, overload with as_text
-    def get_data(self, as_text: bool = ...) -> Any: ...  # returns bytes if as_text is False (the default), else Text
+    @overload
+    def get_data(self, as_text: Literal[False] = ...) -> bytes: ...
+    @overload
+    def get_data(self, as_text: Literal[True]) -> Text: ...
```
Comment (JelleZijlstra) [inline third_party/2and3/werkzeug/wrappers.pyi]: Here too

File: stdlib/3/_winapi.pyi (hunk 1 above)
Comment (srittau) [reply stdlib/3/_winapi.pyi]: For some reason I was assuming, type checkers would infer a proper union type. Seems I was wrong.

**Pull request merged**